guides/common/modules/proc_upgrading-hosts-to-next-major-rhel-release.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Upgrading_Hosts_to_Next_Major RHEL_release_{context}"]
+= Upgrading Hosts to Next Major {RHEL} Release
+
+You can use a job template to upgrade your {RHEL} hosts to the next major version.
+Below upgrade paths are possible:
+
+* {RHEL} 7 to {RHEL} 8
+* {RHEL} 8 to {RHEL} 9
+.Prerequisites
+* Ensure that your {RHEL} hosts meet the requirements for the upgrade.
+ifndef::orcharhino[]
+** For {RHEL} 7 to {RHEL} 8 upgrade, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/planning-an-upgrade_upgrading-from-rhel-7-to-rhel-8[Planning an upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
```

Comment:
```suggestion
** For {RHEL} 7 to {RHEL} 8 upgrade, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/planning-an-upgrade_upgrading-from-rhel-7-to-rhel-8[Planning an upgrade] in _Upgrading from RHEL 7 to RHEL 8_.
```

I feel like this would be match the current guide title usage. My suggestion, if applied, would also affect all other links similarly.